slasher/service/data_update.go

```diff
@@ -37,3 +46,134 @@ func (s *Service) finalisedChangeUpdater() error {
 		}
 	}
 }
+
+func (s *Service) slasherOldAtetstationFeeder() error {
+	if s.beaconClient == nil {
+		return fmt.Errorf("can't feed old attestations to slasher. beacon client has not been started")
+	}
+	ch, err := s.beaconClient.GetChainHead(s.context, &ptypes.Empty{})
+	if err != nil {
+		log.Error(err)
+		return err
+	}
+	if ch.FinalizedEpoch < 2 {
+		log.Info("archive node doesnt have historic data for slasher to proccess. finalized epoch: %d", ch.FinalizedEpoch)
+	}
+	log.Infof("Finalized epoch is currently: %d", ch.FinalizedEpoch)
+	errOut := make(chan error)
+	var errorWg sync.WaitGroup
```

Comment:
What's the point of this errWg and errOut if they're never used? There are no goroutines inside this function so there is no need for either of them.

Reply:
removed. thanks